tests/Application/templates/bundles/SyliusShopBundle/Contact/request.html.twig

```diff
@@ -15,4 +15,11 @@
 
 {% block javascripts %}
     <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
```

Comment:
Why is it here? You already dealt with this in the previous PR, you should rebase this PR on last PR you don't have to deal with it anymore.

Reply:
DONE